amundsen_application/static/js/ducks/notification/reducer.ts

```diff
@@ -34,22 +34,31 @@ export function submitNotificationSuccess(): SubmitNotificationResponse {
 export function closeRequestDescriptionDialog(): ToggleRequestAction {
   return {
     type: ToggleRequest.CLOSE,
+    payload: {
+      checkedInputs: [],
```

Comment:
Just so that the reducer logic for both cases can be `(<ToggleRequestAction>action).payload.checkedInputs`.

Reply:
The alternative is to assign the default value `[]` in the reducer, but remember at one point we had a mix of all the different locations where we would set the default. When we did that `ducks` cleanup a while back, we started placing default values that didn't need to be based off some api call into the  action. See the majority of our `getSomethingFailure()` actions.

Reply:
Could we split `ToggleRequestAction` into separate `OpenRequestAction` and `CloseRequestAction` types?

Reply:
I'd actually disagree, because of the following concepts:
1. One of the benefits of Redux is that different components can call the same action if it makes sense for them to do so.
2. I think it makes sense for them to do so because if we abstract the React part of this away the effect of the action(s) is that same piece of application state gets modified with information from payloads that have the same shape/parameters. Based on that I don't see a need to separate the types based on how the UI happens to be designed.

Reply:
Working through the code if we want to pass the optional column information, this would only be passed on the payload of the open request. Now that the payloads will be different I see a reason to make them actions of different types.

Reply:
The close request action also no longer has a payload.